python/src/scipp/plotting/model2d.py

```diff
@@ -147,8 +158,8 @@ def update_profile(self,
         # Find indices of pixel where cursor lies
         dimx = self.displayed_dims['x']
         dimy = self.displayed_dims['y']
-        x = self._model.data.meta[dimx]
-        y = self._model.data.meta[dimy]
+        x = self._model.data.coords[dimx]
+        y = self._model.data.coords[dimy]
```

Comment:
Why not `meta`?

Reply:
Because I think by construction there should only be coords, no attrs, because we pick and choose only the coords we need when we first create the plot.
It used to be `meta` because `PlotArray` was looking at the original object, and this had to search both coords and attrs.
Now, we make a new DataArray with a new set of coords.

That said, I don't mind reverting to `meta` to minimize the number of changes.

Reply:
I'd suggest reverting to `meta`, given that it may not be unlikely that we may not mess with the coords in the future before this code is reached.